### ISSUE TITLE
70 pixels wasting (unused) space at the bottom

### ISSUE THREAD

**Issue by andrei-korshikov** [mattieserver/netbox-topology-views#406]
### NetBox version

v3.6.3

### Topology Views version

v3.8.0

### Steps to Reproduce

1. Look at the bottom:)

### Expected Behavior

Network diagram should use all the space from top (filter labels) to bottom (status bar icons).

### Observed Behavior

There is 70 pixels gap between diagram area and the top of the status bar icons. This margin is too huge and not in line with three other ones.

![Screenshot_20231014_042537](https://github.com/mattieserver/netbox-topology-views/assets/15959858/a6b86aad-24db-4978-b710-59d7b88842b7)

**Browser:** Chromium Version 118.0.5993.70 (Official Build) Arch Linux (64-bit).
**Resolution:** 1920x1200.

**Comment by mattieserver** [comment]
We have to set the height of the div where the topology is rendered.  Because otherwise the topology would be only a few pixels. The current value is about 67 vh, which works for 16:9 displays.

I really don't know much about css but we should probably have to set a different vh based on the aspect ratio.

**Comment by dreng** [comment]
The CSS in NetBox is done using Bootstrap. Therefore, it might be better to know Bootstrap in particular and not CSS in general.

I have played around a bit with the CSS and Bootstrap, but have not come to any conclusion. Unfortunately, we can't manipulate everything because the basic framework is determined by NetBox. I did not manage to dynamically use all the remaining space available. I am also only somewhat familiar with CSS and only superficially with Bootstrap.

It would be great if someone could step in here.
